addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -457,11 +483,11 @@ def create_employee_from_applicant(self):
                     'work_phone': applicant.department_id and applicant.department_id.company_id
                             and applicant.department_id.company_id.phone or False})
                 applicant.write({'emp_id': employee.id})
-                applicant.job_id.message_post(
-                    body=_('New Employee %s Hired') % applicant.partner_name if applicant.partner_name else applicant.name,
-                    subtype="hr_recruitment.mt_job_applicant_hired")
```

Comment:
``subtype`` is not a valid argument, leading to a warning on runbot.

The correct way to handle this is 
```python
subtype_id=self.env.ref("hr_recruitment.mt_applicant_hired").id
```

``message_post_with_view`` has not the same signature than ``message_post`` by the way.

Reply:
Wrong PR, my bad.